ibm/service/power/resource_ibm_pi_workspace.go

```diff
@@ -158,10 +168,10 @@ func resourceIBMPIWorkspaceDelete(ctx context.Context, d *schema.ResourceData, m
 	return nil
 }
 
-func waitForResourceInstanceDelete(ctx context.Context, client *st.IBMPIWorkspacesClient, id string, timeout time.Duration) (interface{}, error) {
-	stateConf := &resource.StateChangeConf{
-		Pending:    []string{"in progress", "inactive", "active"},
-		Target:     []string{"removed", "pending_reclamation"},
+func waitForResourceInstanceDelete(ctx context.Context, client *instance.IBMPIWorkspacesClient, id string, timeout time.Duration) (interface{}, error) {
+	stateConf := &retry.StateChangeConf{
+		Pending:    []string{State_InProgress, State_Inactive, State_Active},
+		Target:     []string{State_Removed, State_PendingReclaimation},
```

Comment:
Could you verify in the constant State_PendingReclaimation value contain reclaim or reclam? This should be part of the swagger json or please verify it with the backend team. Correct it if needed.

Reply:
```
state
string
The current state of the instance. For example, if the instance is deleted, it will return removed.

Possible values: [active,inactive,removed,pending_removal,pending_reclamation,failed,provisioning,pre_provisioning]
```
It is reclamation, there is a typo in the constant but it has the right values. 
`State_PendingReclaimation = "pending_reclamation"`

[Ref](https://cloud.ibm.com/apidocs/resource-controller/resource-controller#update-resource-instance-response)

It is updated.